api_test.go

```diff
@@ -599,6 +599,11 @@ func TestGroupResetHandler(t *testing.T) {
 				signalChan <- false
 			}
 		})
+		if err != nil {
+			t.Log(err)
+			t.Fail()
```

Comment:
You mean `Fatal`, which can't be run from a separate goroutine. I need to log the error, mark the test as failed, and make the main goroutine stop, which is what this does.